### PR TITLE
skip demo deploy

### PR DESCRIPTION
**Pull request by Albert-Hugo**
Skip  the demo app to deploy. 

No.

Add skip plugin to maven pom file.

**Checklist**:

- [x] Documentation added
      
    Yes
- [x] Changelog updated
      
- [x] Tests added
      
     Yes
- [x] Ready to be merged
      
     Yes

### CODE REVIEW

**Comment by Albert-Hugo** [comment]
> Hi @Albert-Hugo
> 
> Congratulations on your first contribution 🥳 Thank you! - would you add a [changelog entry](https://github.com/codecentric/chaos-monkey-spring-boot/blob/main/chaos-monkey-docs/src/main/asciidoc/changes.adoc) for this change? :)
> 
> Change looks good 👍

Hi @WtfJoke , I had updated the changelog entry. Could you please help to have a look? Thanks.

File: chaos-monkey-docs/src/main/asciidoc/changes.adoc [hunk 1]
```diff
@@ -1,17 +1,19 @@
 [[changes]]
-== Changes in {project-version}
+== Changes in 2.4.6-SNAPSHOT
```
Comment (WtfJoke) [inline chaos-monkey-docs/src/main/asciidoc/changes.adoc:2]: That will be done automatically, please undo that

File: chaos-monkey-docs/src/main/asciidoc/changes.adoc [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
 
 === Bug Fixes
 // - https://github.com/codecentric/chaos-monkey-spring-boot/pull/xxx[#xxx] Added example entry. Please don't remove.
 
 === Improvements
 // - https://github.com/codecentric/chaos-monkey-spring-boot/pull/xxx[#xxx] Added example entry. Please don't remove.
+https://github.com/codecentric/chaos-monkey-spring-boot/pull/227[#224]
```
Comment (WtfJoke) [inline chaos-monkey-docs/src/main/asciidoc/changes.adoc:9]:
```suggestion
- https://github.com/codecentric/chaos-monkey-spring-boot/pull/227[#227] Fix #224 by not deploying demo projects to maven central
```

File: chaos-monkey-docs/src/main/asciidoc/changes.adoc [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
 
 === New Features
 // - https://github.com/codecentric/chaos-monkey-spring-boot/pull/xxx[#xxx] Added example entry. Please don't remove.
 
 === Contributors
 This release was only possible because of these great humans:
+Albert-Hugo
```
Comment (WtfJoke) [inline chaos-monkey-docs/src/main/asciidoc/changes.adoc:16]:
```suggestion
- https://github.com/Albert-Hugo[@Albert-Hugo]
```

Like in the example :)

**Review by WtfJoke — COMMENTED** [review]
@Albert-Hugo Almost there, I made some suggestions. You can apply them, then I'll merge it :)

**Comment by Albert-Hugo** [comment]
> @Albert-Hugo Almost there, I made some suggestions. You can apply them, then I'll merge it :) (you can add them to batch and then commit through github or make the changes manually on your own)

Got it. Thank you.

File: chaos-monkey-docs/src/main/asciidoc/changes.adoc (hunk 1 above)
Comment (WtfJoke) [inline chaos-monkey-docs/src/main/asciidoc/changes.adoc:2]:
```suggestion
== Changes in {project-version}
```

**Comment by WtfJoke** [comment]
Looks good :) LGTM ✈️

**Review by WtfJoke — APPROVED** [review]

**Pull request merged**